apiserver/src/main.rs

```diff
@@ -1,3 +1,33 @@
-fn main() {
-    println!("Hello, world!");
+use apiserver::api::{self, APIServerSettings};
+use apiserver::error::{self, Result};
+use models::node::K8SBottlerocketNodeClient;
+
+use snafu::ResultExt;
+
+use std::fs;
+
+#[actix_web::main]
+async fn main() {
+    match run_server().await {
+        Err(error) => {
+            fs::write("/dev/termination-log", format!("{}", error))
```

Comment:
Nit: Should `/dev/termination-log` be a constant?

Also, I'm confused, why is this error written to `/dev/termination-log`, but the server errors to whatever the environment variables have?